runtime/binding-tcp/src/test/java/io/aklivity/zilla/runtime/binding/tcp/internal/streams/ClientIT.java

```diff
@@ -209,20 +207,6 @@ public void shouldEstablishConnectionIPv6() throws Exception
         k3po.finish();
     }
 
-    @Test
-    @Configuration("client.host.yaml")
-    @Specification({
-        "${app}/connection.failed/client"
-    })
-    public void connnectionFailed() throws Exception
-    {
-        k3po.finish();
-        Thread.sleep(250); // TODO: engine quiesce instead of close
-        EngineStats stats = engine.stats("test", "app0");
-        fullFence();
-        assertEquals(1, stats.initialErrors());
-    }
-
```

Comment:
I think we can keep this test, just without the error counter check, agree?

Reply:
done